.gitignore

```diff
@@ -16,6 +16,7 @@ config.log
 config.status
 config.sub
 compile
+core
```

Comment:
We don't need to ignore core files. They're not part of the standard SIPp compilation cycle, I hope ;P

Please drop this from the changeset.